templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -166,6 +166,22 @@
 						{{svg "octicon-milestone" 18 "gt-mr-3"}}
 						{{.Issue.Milestone.Name}}
 					</a>
+					{{if .Issue.Milestone.Labels}}
+						<div class="ui labels list">
+							<span class="labels-list">
```

Comment:
IIRC, there is `RenderLabel` / `RenderLabels` in `helper.go`, other label pages use these functions?